client/my-sites/stats/activity-log-banner/progress-banner.jsx

```diff
@@ -11,31 +11,44 @@ import ActivityLogBanner from './index';
 import ProgressBar from 'components/progress-bar';
 
 function SuccessBanner( {
-	translate,
 	moment,
+	percent,
+	status,
+	timestamp,
+	translate,
 } ) {
-	// FIXME: real dates
-	const date = 1496400468285;
-	// FIXME: real progress
-	const progress = 25;
-
 	return (
 		<ActivityLogBanner
 			status="info"
 			title={ translate( 'Currently restoring your site' ) }
 		>
 			<p>{ translate(
-				'We\'re in the process of restoring your site back to %s.' +
-				'You\'ll be notified once it\'s complete.',
-				{ args: moment( date ).format( 'LLLL' ) }
+				"We're in the process of restoring your site back to %s. " +
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'We're in the process of restoring your site back to %s.You'll be notified once it's complete.' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).